td-agent/yum/td-agent.spec.in

```diff
@@ -101,7 +108,14 @@ fi
 
 %files
 %defattr(-,root,root,-)
-/opt/*
+/opt/td-agent/LICENSES/*
+/opt/td-agent/bin/*
+/opt/td-agent/lib/ruby/*
+/opt/td-agent/share/*
+/opt/td-agent/lib/libruby.so*
+/opt/td-agent/lib/libjemalloc.so*
+/opt/td-agent/lib/libjemalloc*.a
```

Comment:
libjemalloc*.a isn't used, we can drop it.
(It should be moved to `td-agent-static` package if we keep the file.)

Reply:
It will be removed by #76.